RELEASE.md

```diff
@@ -1,3 +1,63 @@
+# Release 1.7.0
+
+## Major Features And Improvements
+* Eager mode is moving out of contrib, try `tf.enable_eager_execution()`.
+* Graph rewrites emulating fixed-point quantization compatible with TensorFlow Lite, supported by new `tf.contrib.quantize` package.
+* Easily customize gradient computation with `tf.custom_gradient`.
+* TensorBoard Debugger Plugin, tfdbg: TensorFlow Debugger's graphical user interface (GUI), is now in alpha.
```

Comment:
s/tfdbg: TensorFlow Debugger's graphical user interface (GUI)/The graphical user interface (GUI) of TensorFlow Debugger (tfdbg)/